core/fortran-example/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+option(Cabana_ENABLE_FORTRAN_EXAMPLES "Build Fortran examples" OFF)
+if(NOT Cabana_ENABLE_FORTRAN_EXAMPLES)
+  return()
```

Comment:
`else() enable_language(FORTRAN)`

Reply:
Just conditionally add the directory
```CMake
if(Cabana_ENABLE_FORTRAN_EXAMPLES)
  add_subdirectory(tutorial)
endif()
```

Reply:
@junghans I got errors after suggested changes:
```CMake Error: Could not find cmake module file: CMakeDetermineFORTRANCompiler.cmake
CMake Error: Error required internal CMake variable not set, cmake may not be built correctly.
Missing variable is:
CMAKE_FORTRAN_COMPILER_ENV_VAR
CMake Error: Error required internal CMake variable not set, cmake may not be built correctly.
Missing variable is:
CMAKE_FORTRAN_COMPILER
CMake Error: Could not find cmake module file: /home/gchen/Research/fortran-ex/06/Cabana/build-gpu/CMakeFiles/3.12.1/CMakeFORTRANCompiler.cmake
CMake Error at core/fortran-example/CMakeLists.txt:5 (enable_language):
  No CMAKE_FORTRAN_COMPILER could be found.

  Tell CMake where to find the compiler by setting the CMake cache entry
  CMAKE_FORTRAN_COMPILER to the full path to the compiler, or to the compiler
  name if it is in the PATH.


CMake Error: Could not find cmake module file: CMakeFORTRANInformation.cmake
CMake Error: CMAKE_FORTRAN_COMPILER not set, after EnableLanguage
-- Configuring incomplete, errors occurred!
```

Reply:
Sorry

Reply:
Sorry

Reply:
Drop, and invert the `if` clause.

Reply:
Please invert the logic